sections/3-coordinates.tex

```diff
@@ -32,54 +31,60 @@ \section{Solar Coordinates}
 \code{get\_horizons\_coord} allows one to query JPL HORIZONS\footnote{\url{https://ssd.jpl.nasa.gov/?horizon}} for the location of a wide range of solar-system bodies.
 JPL HORIZONS includes ephemeris information not only for planets and other natural bodies in the solar system, but also for major spacecraft such as SDO and SOHO.
-This function requires the \package{astroquery}\footnote{\url{https://astroquery.readthedocs.io/en/latest/}} package to be installed and an Internet connection.
+This function requires the \package{astroquery} package \citep{ginsburg_astroquery_2019} to be installed and an Internet connection.
 
 The coordinate framework is also used to calculate Sun-specific coordinate information with high accuracy.
 These functions are under \package{sunpy.coordinates.sun} and return values such as Carrington rotation number for the provided time.
 Nearly all of the returned values match values in the \textit{Astronomical Almanac} to published precision (e.g., the hundredth of an arcsecond for apparent right ascension).
 For times that are provided to these functions, one should take care to specify whether the time scale is UT or some other time scale (e.g., TT).
 
-\subsection{Solar differential coordination - Jack}
+\subsection{Differential Rotation}
+\label{sec:differential_rotation}
+
+%1-2 sentences intro; 1-2 sentences describing what is new; 1 sentence that acknowledges the figure.
 
-\todo{This section}
-One paragraph long.
-Not clear if it needs to be in this location.
-Should it be in a map utility section?
+% Can delete the first two sentences if the background info feels unnecessary 
 
-Don't forget to refer to the figure!
-An important application of....
-Figure \ref{fig:diff_rot}
+When analyzing the dynamics of observed solar emission, it is important to account for variations due to the rotation of the Sun.
+In particular, one must account for \textit{differential rotation}, or the latitudinally-varying rotation rate due to the non-rigidity of the solar interior.
+The \package{sunpy.physics.differential\_rotation} subpackage provides tools for transforming both \code{SkyCoord} coordinate and \code{Map} objects according to the differential rotation of the Sun.
+The amount of rotation can be specified using either a new time, a time interval or a new observer.
 
+While earlier versions of \sunpy included tools for dealing with differential rotation, this release includes several improvements.
+For example, applying the effect of solar differential rotation now properly accounts for the changing position of the observer.
+This can be important because observers on the Earth must take into account the solar differential rotation as well as the motion of the Earth when calculating a location on the Sun.
```

Comment:
```suggestion
This can be important because, for example, observers on the Earth must take into account the motion of the Earth in addition to solar differential rotation when calculating how much a location on the Sun appears to move.
```